Game/Source/GDKTestGyms/BenchmarkGymGameModeBase.h

```diff
@@ -156,9 +157,9 @@ class GDKTESTGYMS_API ABenchmarkGymGameModeBase : public AGameModeBase
 	void SetLifetime(int32 Lifetime);
 	int32 GetPlayerControllerCount() const;
 #if	STATS
-	void SetStatTimer(const FString& TimeString);
-	void InitMemReportTimer(const FString& MemReportIntervalString);
+	void InitStatTimer(const FString& CPUProfileString);
 #endif
+	void InitMemReportTimer(const FString& MemReportIntervalString);
```

Comment:
This is only implemented when `STATS` is enabled, did you also mean to move the implementation out of an #if block?

Reply:
I can only see usage of this inside `STATS` though